packages/coreutils/src/pageconfig.ts

```diff
@@ -82,6 +82,21 @@ namespace PageConfig {
     }
     return configData[name] || '';
   }
+  
+  /**
```

Comment:
@lucbouchard1, it can help to comment out all but [this line](https://github.com/jupyterlab/jupyterlab/blob/0e807861c6957cf4f208d2ab0a79d51912aed7cc/test/src/index.ts#L47) and [this line](https://github.com/jupyterlab/jupyterlab/blob/0e807861c6957cf4f208d2ab0a79d51912aed7cc/test/src/index.ts#L4) to run just the module under test.